app/workers/audio_callback_worker.rb

```diff
@@ -14,7 +14,7 @@ def perform(_sqs_msg, job)
     audio.frequency = (job['frequency'] || 0) / 1000.0
 
     # decode content type and mpeg layer from basic "format" string
-    mime_types = MIME::Types.type_for(job['format']).map(&:to_s)
+    mime_types = MIME::Types.type_for(job['format'] || '').map(&:to_s)
```

Comment:
For some reason, I was thinking that `type_for` would handle the `nil`.  But this is better.